src/helpers/register-plugin-routes.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash')
+const composeRouteString = require('./compose-route-string')
+
+function registerProviderRoutes ({ provider, controller, server, pluginRoutes }, options = {}) {
+  const namespace = provider.namespace.replace(/\s/g, '').toLowerCase()
+  const { hosts, disableIdParam } = provider
+  const { routePrefix } = options
+  const pluginRouteMap = {}
+
+  pluginRoutes.forEach(route => {
+    const { path, methods, absolutePath, output } = route
```

Comment:
Can you lowercase/uppercase the method names here to address #85?

Reply:
Handled in 289d413.